From fbfa25edd38f278e0d4056699fa0e0eddc94f107 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 17 Aug 2018 08:53:17 +0300
Subject: [PATCH] BTL [pcie phy]: restore disable of equalization redo

SNSP approve this as a ligitimate approach to enable
the link bring up with XEON Root Complex
---

diff --git a/board/habanalabs/common/btl/pcie_phy.s b/board/habanalabs/common/btl/pcie_phy.s
index d916052..4322306 100644
--- a/board/habanalabs/common/btl/pcie_phy.s
+++ b/board/habanalabs/common/btl/pcie_phy.s
@@ -40,12 +40,8 @@
 #endif /* DEV_IS_DALI */
 cfg mmPCIE_PHY_PHY_EXT_CTRL_SEL, 0x00000001
 
-/*
- * disable the EQ REDO
- * Currently, we are not certain this is good or bad...
- * Meanwhile leave this out.
- */
-/* cfg mmPCIE_DBI_GEN3_RELATED_OFF, 0x00402801 */
+/*Disable the EQ REDO */
+cfg mmPCIE_DBI_GEN3_RELATED_OFF, 0x00402801
 
 /*
  * enable direction of change,
